application/listeners/listeners.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/silinternational/cover-api/models"
 )
 
+const NotifierKey = "notifier"
```

Comment:
```suggestion
const EventPayloadNotifier = "notifier"
```
for consistency with `EventPayloadID`

Reply:
:+1: